index.html

```diff
@@ -23,7 +23,7 @@
     </nav>
   </header>
   <section class="firstPage">
-    <h1 class="heyHead">Hey There. I'm Alex</h1>
+    <h1 class="heyHead">Hey There. I'm Hamza</h1>
```

Comment:
- [ ] It is a better practice to maintain a meaningful hierarchy in your titles. Only keep this as your `h1` tag. The other section titles should have `h2` tags and project titles should have `h3` tags.